pkg/cloudprovider/amifamily/ami.go

```diff
@@ -193,23 +193,41 @@ func (p *AMIProvider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[stri
 	return output.Images, nil
 }
 
-func getFilters(amiSelector map[string]string) []*ec2.Filter {
+func getFiltersAndOwners(amiSelector map[string]string) ([]*ec2.Filter, []*string) {
 	filters := []*ec2.Filter{}
+	owners := []*string{}
 	for key, value := range amiSelector {
-		if key == "aws-ids" {
+		switch key {
+		case "aws-ids":
```

Comment:
Can you make it so that aws::ids and aws-ids are both special cased so that we don't break users?
```suggestion
		case "aws-ids", "aws::ids":
```